README.md

```diff
@@ -86,6 +86,7 @@ There is a bit of manual effort necessary from year to year. These tasks include
 * update `$shorten_topics` in `scroll.php` to reflect the updated track list, matching exactly the keys to what is being supplied by the xml from drupal minus spaces
 * update `$shorten_topics` in `room.php` to match `scroll.php`
 * create a style for each of the keys in `$shorten_topics` in `style.css` matching it to the public site
+* add colors for topics/tracks in `style.css` 
```

Comment:
Thanks for adding this to the list!

Reply:
Ok, I removed the special char @ from two files:

https://github.com/carlynlee/scale-signs/blob/shorten_topics_tracklist/server/scroll.php#L50

https://github.com/carlynlee/scale-signs/blob/shorten_topics_tracklist/server/style.css#L561

I rebuilt the image and tried to view the track listing again but I'm unsure why the colors aren't applied:
<img width="1368" alt="Screenshot 2024-03-04 at 3 31 16 PM" src="https://github.com/socallinuxexpo/scale-signs/assets/916785/3b3eeb4c-b7b0-498f-bf91-82277d659843">

Reply:
did you rebuild the container? It looks like its still the old name `Foss @ Home` from your screenshot (which it should be `FOSS HOME` from your diff)

BTW keep the `@` on the right side of the assignment. So the entry in `scroll.php` would look like:

```
"FOSSHOME"	=>	 "FOSS @ HOME",
```

Reply:
yep, I ran `docker-compose build --no-cache` and then `docker-compose up -d`